### DISCUSSION TITLE
Additional Dataflow Detail?

### DISCUSSION THREAD

**Discussion by GrimnebulinAhab** [migueesc123/PowerBIRESTAPI#159] [Q&A] [answered]
The Scanner API returns new information that is not currently present in the custom connector. For example, dataflow refresh schedules. I know how to use the GetData function provided in the custom connector, but it doesn't look like I can go about it this way, and I don't see a way to look for additional data provided in the results. Any information you have is greatly appreciated.

**Comment by klinejordan** [comment, marked as answer]
While there is a function that allows you to call the Scanner API for up to 100 workspaces at a time (see below and the attached example of how to use it), we cannot use the Scanner API for the standard admin queries because it involves API polling and M is not a good language for that. To use the Scanner API to its fullest extent you really need to design your own ETL to process the results from the scanner API. 
![image](https://user-images.githubusercontent.com/48561922/232949577-76b51e68-08bd-4c9e-b870-7e5cdeeca5f0.png)
[Scanner API Example.zip](https://github.com/migueesc123/PowerBIRESTAPI/files/11267915/Scanner.API.Example.zip)

**Comment by GrimnebulinAhab** [comment]
Thank you for the quick response!
